chainstate/test-suite/src/tests/syncing_tests.rs

```diff
@@ -131,10 +128,7 @@ fn get_headers(#[case] seed: Seed) {
 
         // The locator is from this exact chain, so `get_headers` should return an empty sequence.
         let locator = tf.chainstate.get_locator().unwrap();
-        assert_eq!(
-            tf.chainstate.get_headers(locator.clone(), header_limit).unwrap(),
-            vec![],
-        );
+        assert_eq!(tf.chainstate.get_headers(locator.clone(), header_limit).unwrap(), vec![],);
```

Comment:
Quite annoying that the formatter does not handle trailing commas in macros. At least for some known ones where it's expected to be fine.